resources/function-controller/charts/docker-registry/templates/deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
 {{ toYaml .Values.podLabels | indent 8 }}
         {{- end }}
       annotations:
-        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
![image](https://user-images.githubusercontent.com/17271979/75625342-85cd6b00-5bbd-11ea-9a81-491ee9c23fb7.png)

Reply:
You can also use timestamp :)